data/human/fleets.txt

```diff
@@ -1971,6 +1976,9 @@ fleet "Large Southern Pirates"
 		"Falcon (Heavy)"
 	variant 1
 		"Falcon (Laser)"
+	variant 1
+		"Falcon"
```

Comment:
What about a faster escort of some (2-3?) ravens? A falcon, whilst ok fast, would slow this one down I think?

Reply:
Ravens are northern, we're here in the south. Not everything need to be hyperspeed.

Reply:
I guess it's ok caus the falcon could be participating in the eventual planetary assault too